pub-critic/src/app/components/sideMenu/SideMenu.tsx

```diff
@@ -5,19 +5,24 @@ import classes from "./SideMenu.module.scss";
 interface SideMenuProps {
   toggleMenu: () => void;
   active: boolean;
+  user?: boolean;
 }
 
-export const SideMenu = ({ toggleMenu, active }: SideMenuProps) => {
+export const SideMenu = ({ toggleMenu, active, user }: SideMenuProps) => {
   return (
-    <div className={classes.sideMenu}>
+    <div
+      className={`${classes.sideMenu} ${active ? classes.active : ""}`}
```

Comment:
https://github.com/lukeed/clsx

Reply:
yep gonna use clsx in the future for this, might change it later if I have time